optashift-employee-rostering-shared/src/main/java/org/optaplanner/openshift/employeerostering/shared/roster/view/AbstractRosterView.java

```diff
@@ -31,8 +30,8 @@
     protected List<Spot> spotList;
     @NotNull
     protected List<Employee> employeeList;
-    @NotNull
-    protected List<TimeSlot> timeSlotList;
+
+    protected RosterState rosterState;
```

Comment:
I'd add a NotNull on this one too, as a signal that it's never null.

But views are never saved, so maybe we should simply remove all javax.validation annotations on views?

In any case: either we add this one xor we remove all of them. the worst case is mixing: doing some, but not all.